libs/components/data-manager/src/lib/modules/data-manager/models/data-manager-column-picker-sort-strategy.ts

```diff
@@ -1,4 +1,14 @@
+/**
+ * These options specify the sorting applied to columns.
+ */
 export enum SkyDataManagerColumnPickerSortStrategy {
+  /**
+   * No sorting is applied to the columns.
```

Comment:
Can we start this with a verb like the other descriptions? Or does that change the meaning?
```suggestion
   * Applies no sorting to the columns.
```